easyaudit/migrations/0009_auto_20180314_2225.py

```diff
@@ -20,6 +20,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='requestevent',
             name='url',
-            field=models.TextField(db_index=True),
+            field=models.TextField(db_index=False),
```

Comment:
this migration actually failed for us. So I'm hoping that users whom have not yet run the migration won't be affected by it, due to this change.

Any users that want to remove the index will have to do it by hand, via raw SQL.

Reply:
Are you using MySQL? If so, keeping the index, changing the column type to `CharField`, and adding a `max_length` attribute did the trick for me.

Reply:
Yes, using MySQL. Interesting. I could see us wanting to keep as a TextField though, so there is no information loss.

On a separate PR I had suggested a url field and params field.

I don't have the luxury at this time to update the field on the fly (we are in production with this code change already). But what do you think about splitting the fields up?